app/models/case_log.rb

```diff
@@ -254,15 +256,12 @@ def get_totadult
     end
   end
 
-  def get_la(postcode)
+  def get_inferred_la(postcode)
     postcode_lookup = nil
     Timeout.timeout(5) { postcode_lookup = PIO.lookup(postcode) }
```

Comment:
Am I right in thinking this API call still happens on every `save!`? Might be better to check whether `postcode` is a field that's changed as a condition for sending it?

Reply:
This private method only gets called when this condition is false:
`property_postcode.blank? || postcode_known == "No"`

So yeah we end up making a call every time the field is saved with a postcode populated.
Is there an easy way in AR to detect that an attribute has changed (other than manually loading from DB)?

Reply:
Yes you can do something like:

```
before_save :infer_postcode, if: postcode_changed?
```